File: 10_week_2/arr_concat.py
def arr_concat(arr1, arr2):
	new_arr = []

	for val in arr1:
		new_arr.append(val)
	for val in arr2:
		new_arr.append(val)

	return new_arr


arr1 = [1, 2]
arr2 = [3, 4]

new_list = arr_concat(arr1, arr2)
print(new_list)

File: 10_week_2/arr_concat.js
function arr_concat(arr1, arr2) {

	var new_arr = [];

	for (var i = 0, j = arr1.length; i < j; i++) {
		new_arr.push(arr1[i]);
	}
	for (var i = 0, j = arr2.length; i < j; i++) {
		new_arr.push(arr2[i]);
	}

	return new_arr;
}

var arr1 = [1, 2];
var arr2 = [3, 4];

var new_array = arr_concat(arr1, arr2);
console.log(new_array);
